templates/eslintrc

```diff
@@ -65,7 +63,9 @@
     "consistent-return": 2,
     "curly": [2, "all"],
     "default-case": 2,
-    "dot-notation": [2, {"allowKeywords": true}],
+    "dot-notation": [0, {
```

Comment:
Oh. Must be slipped through. Will revert this.

On 11 August 2015 at 17:40, Robert Katzki <notifications@github.com> wrote:

> In templates/eslintrc
> <https://github.com/ubilabs/project-template/pull/20#discussion_r36760211>
> :
>
> > @@ -65,7 +63,9 @@
> >      "consistent-return": 2,
> >      "curly": [2, "all"],
> >      "default-case": 2,
> > -    "dot-notation": [2, {"allowKeywords": true}],
> > +    "dot-notation": [0, {
>
> Why that change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubilabs/project-template/pull/20/files#r36760211>.
>



-- 
--
Frank Mecklenburg
mecklenburg@ubilabs.net

Ubilabs GmbH
Juliusstr. 25
22769 Hamburg, Germany

Telefon +49 40 - 60 94 661 - 0
Fax +49 40 - 60 94 661 - 66

Handelsregister Hamburg: HRB 110269
Steuernummer: 46/765/01253


Geschäftsführer: Michael Pletziger, Jens Wille

--